src/harness/harness.c

```diff
@@ -264,13 +268,34 @@ void Harness_Hook_BrBufferUpdate(br_pixelmap* pm, br_token use, br_uint_16 flags
     }
 }
 
+// Input hooks
+void Harness_Hook_GetMousePosition(int* pX, int* pY) {
+    Input_GetMousePosition(pX, pY);
+}
+
+void Harness_Hook_GetMouseButtons(int* pButton1, int* pButton2) {
+    Input_GetMouseButtons(pButton1, pButton2);
+}
+
+// Window hooks
+void Harness_Hook_GetWindowSize(int* pW, int* pH) {
+    Window_GetSize(pW, pH);
+}
+
+// Sound hooks
 void Harness_Hook_S3Service(int unk1, int unk2) {
     Sound_Service();
 }
 
 void Harness_Hook_S3StopAllOutletSounds() {
 }
 
-void Harness_Hook_FlushRenderer() {
-    renderer->FlushBuffers(last_colour_buffer, last_depth_buffer);
+// Various hooks
+void Harness_Hook_Delay_ms(int ms) {
+    Time_Delay_ms(ms);
+}
```

Comment:
I noticed there is also `void OS_Sleep(int delay_ms)`.

Reply:
yes, lets use `OS_Sleep` instead. Also, I don't think we need to have a harness_hook to front an `OS_*` call - we can call `Harness_*` or `OS_*` functions direct from OG code